pkg/controller/clusters/validations.go

```diff
@@ -54,3 +57,57 @@ func (v *ClusterValidator) ValidateManagementClusterName(ctx context.Context, lo
 
 	return nil
 }
+
+// ValidateManagementEksaVersion checks if the workload cluster's EksaVersion does not exceed its management cluster.
+func (v *ClusterValidator) ValidateManagementEksaVersion(ctx context.Context, log logr.Logger, cluster *anywherev1.Cluster) error {
+	mgmtCluster := &anywherev1.Cluster{}
+	mgmtClusterKey := client.ObjectKey{
+		Namespace: cluster.Namespace,
+		Name:      cluster.Spec.ManagementCluster.Name,
+	}
+
+	if err := v.client.Get(ctx, mgmtClusterKey, mgmtCluster); err != nil {
+		if apierrors.IsNotFound(err) {
+			err := fmt.Errorf("unable to retrieve management cluster %v: %v", cluster.Spec.ManagementCluster.Name, err)
+			log.Error(err, "Invalid cluster configuration")
+			return err
+		}
+	}
+
+	if mgmtCluster.Spec.EksaVersion == nil {
```

Comment:
I think this is valid, right?
while the management cluster is upgraded, this will be nil

Reply:
So I should return nil here? No error?